app/components/provider_interface/application_choice_header_component.rb

```diff
@@ -43,18 +44,18 @@ def deferred_offer_wizard_applicable?
         application_choice.recruitment_cycle == RecruitmentCycle.previous_year
     end
 
-    def deferred_offer_equivalent_course_option_available?
```

Comment:
Removed as we no longer worry about course availability at this stage

Reply:
:+1: Having discussed, this doesn't method isn't called in order to render the details page anymore.